packages/editor/src/store/selectors.js

```diff
@@ -27,18 +27,12 @@ import { createRegistrySelector } from '@wordpress/data';
  * Internal dependencies
  */
 import { PREFERENCES_DEFAULTS } from './defaults';
-import { EDIT_MERGE_PROPERTIES } from './constants';
-
-/***
- * Module constants
- */
-export const POST_UPDATE_TRANSACTION_ID = 'post-update';
-const PERMALINK_POSTNAME_REGEX = /%(?:postname|pagename)%/;
-export const INSERTER_UTILITY_HIGH = 3;
```

Comment:
I guess these were overlooked in the split for block editor? I verified they're not used in editor.

Reply:
Either that or I resolved the merge incorrectly when bringing into this branch.